third-party/realdds/src/dds-serialization.cpp

```diff
@@ -242,4 +350,36 @@ void override_endpoint_qos_from_json( eprosima::fastdds::dds::RTPSEndpointQos &
 }
 
 
+void override_participant_qos_from_json( eprosima::fastdds::dds::DomainParticipantQos & qos, nlohmann::json const & j )
+{
+    if( ! j.is_object() )
+        return;
+    rsutils::json::get_ex( j, "participant-id", &qos.wire_protocol().participant_id );
+    rsutils::json::get_ex( j, "lease-duration", &qos.wire_protocol().builtin.discovery_config.leaseDuration );
+
+    rsutils::json::get_ex( j, "use-builtin-transports", &qos.transport().use_builtin_transports );
+    if( auto udp_j = rsutils::json::nested( j, "udp" ) )
+    {
+        for( auto t : qos.transport().user_transports )
+            if( auto udp_t = std::dynamic_pointer_cast< eprosima::fastdds::rtps::UDPTransportDescriptor >( t ) )
+            {
+                rsutils::json::get_ex( udp_j, "send-buffer-size", &udp_t->sendBufferSize );
```

Comment:
Separate to a function `parse_udp_settings`.
I beleive we will have more settings to deal with in the future

Reply:
Done.